apidef/oas/authentication_test.go

```diff
@@ -93,3 +93,15 @@ func TestSignature(t *testing.T) {
 
 	assert.Equal(t, emptySignature, resultSignature)
 }
+
+func TestBasic(t *testing.T) {
```

Comment:
Looks good! But I would have a test with all fields set.

Reply:
We have that test too but it is for entire apidef. So it is skipped for now as we didn’t implement all conversions.

Reply:
The final goal is to make that test green. Here the test: https://github.com/TykTechnologies/tyk/blob/ab13bdb0d200d9f38c067012be20e365bcc70e0e/apidef/oas/root_test.go#L27